__tests__/app.test.js

```diff
@@ -375,3 +361,44 @@ describe("/api/users", () => {
       });
   });
 });
+
+// Feature Sorting Queries TDD
+describe("/api/articles", () => {
+  test("GET: 200 - responds with articles sorted by votes in descending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(articles).toBeSortedBy("votes", { descending: true });
+      });
```

Comment:
Nice, no reason to check all the articles properties again just check what they are sorted by makes sense